packages/tkg-clusterclass/bundle/config/packageinstalls/tkg-infra-clusterclass-pi.yaml

```diff
@@ -1,74 +0,0 @@
-#@ load("@ytt:overlay", "overlay")
```

Comment:
By removing this: 

Would there still be a possibility to write a single overlay which would get applied for all clusterclasses (without copying it to every clusterclass?)

We make use of that functionality in #3862 🤔